app/javascript/gobierto_dashboards/webapp/lib/utils.js

```diff
@@ -0,0 +1,10 @@
+export function formatCurrency(amount) {
```

Comment:
this file could be move to the vue-filters (`lib/shared`) or even that, check there if there's some common function you may reuse for your purpose

Reply:
yeah, this is already removed in #3057 (https://github.com/PopulateTools/gobierto/pull/3057/commits/faab468f320a668fd523658f9efa1f274ff50393) because it will use an already created filter.